tools/dependency_summary.py

```diff
@@ -671,6 +675,8 @@
 # If we can find one and only one of these files in a package, then we can be confident
 # that it's the intended license text.
 COMMON_LICENSE_FILE_NAME_ROOTS = {
+    "": ["license", "licence", "LICENSE"],
+    "Apache-2.0": ["license-apache", "licence-apache", "LICENSE-APACHE"],
```

Comment:
I'm surprised this was necessary - I thought we had some use of `name.lower()` in the code that uses this in order to deal with capitalization issues, but maybe that's not working quite as intended?

Reply:
Yeah, I'd think that should do the trick too.  Right now, I'm having issues reproducing the problem I saw, so I'll just remove these particular changes and the land the logging stuff without them.